g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/RegExpValidatorUtils.java

```diff
@@ -66,6 +66,14 @@ public static boolean IsLetterAndNumberAndValidchar(String str) {
 		String regex = "^[A-Za-z0-9_\\-\\.]+$";
 		return match(regex, str);
 	}
+	
```

Comment:
Should the old method 'IsLetterAndNumberAndValidchar' be removed?

Reply:
other method still use it